### DISCUSSION TITLE
Use GitHub release so people can subscribe to them

### DISCUSSION THREAD

**Discussion by MoOx** [invertase/react-native-firebase#5382] [Ideas]
Hi!

First, thanks for this library 🙏
Here is a simple feature request: could you transform your git tag as GitHub release so people can watch the repo & get notified when a release comes out?
I have a little trick that I use that make it no-brainer: [](https://www.npmjs.com/package/github-release-from-changelog)

This tool edits the git tag on GitHub and create a GitHub release with the correct changelog section.
In short:
- do you release process as usual
- just execute this tool when release is done, it will look into your package.json for the version, will read your changelog to find the right section to put.

You have a --dry-run option to test this before using it for real.

Note: you only need in addition to this tool a `GITHUB_TOKEN` available as an env var.

**Comment by mikehardy** [comment]
PRs happily accepted, our release script is here: https://github.com/ankidroid/ankidroiddocs/blob/master/.github/workflows/publish.yml

**Reply by MoOx** [reply]
I guess you mean https://github.com/invertase/react-native-firebase/blob/master/.github/workflows/publish.yml ?

**Reply by mikehardy** [reply]
Ha dangit. But please feel free to PR to that one too if you see something :laughing:

**Reply by mikehardy** [reply]
I like the idea btw - I had not even realized we *weren't* tagging them as official releases and I know it's hard to track otherwise
